.github/workflows/release.yml

```diff
@@ -0,0 +1,123 @@
+            --repo ${{ github.repository }} \
+            --title ${{ github.ref_name }} \
+            --notes '' \
+            --draft \
+            ${{ github.ref_name }}
+  build_zips:
+    needs: create_draft_release
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - otp_release: 22
+          - otp_release: 23
+          - otp_release: 24
+            build_docs: build_docs
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - uses: erlef/setup-beam@v1
+        with:
+          # The otp-version is the latest version of the release. It doesn't guarantee
+          # the minor (or patch) version when running this workflow.
+          otp-version: ${{ matrix.otp_release }}
+      - name: Build Elixir Release
+        run: |
+          vsn=$(cat VERSION)
+          make Precompiled.zip
+          mv Precompiled-v${vsn}.zip elixir-otp-${{ matrix.otp_release }}.zip
+          echo "$PWD/bin" >> $GITHUB_PATH
+      - uses: actions/upload-artifact@v2
+        with:
+          name: elixir-otp-${{ matrix.otp_release }}.zip
+          path: elixir-otp-${{ matrix.otp_release }}.zip
+      - uses: actions/checkout@v2
+        with:
+          repository: elixir-lang/ex_doc
+          path: ex_doc
+        if: ${{ matrix.build_docs }}
+      - name: Build ex_doc
+        if: ${{ matrix.build_docs }}
+        run: |
+          mv ex_doc ../ex_doc
+          cd ../ex_doc
+          ../elixir/bin/mix do local.rebar --force, local.hex --force, deps.get, compile
+          cd ../elixir
+      - name: Build Docs
+        if: ${{ matrix.build_docs }}
+        run: |
+          vsn=$(cat VERSION)
+          make Docs.zip
+          mv Docs-v${vsn}.zip Docs.zip
+      - uses: actions/upload-artifact@v2
+        with:
+          name: Docs.zip
+          path: Docs.zip
+        if: ${{ matrix.build_docs }}
+  upload_assets:
+    needs: build_zips
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - uses: actions/download-artifact@v2
+        with:
+          name: elixir-otp-22.zip
+      - uses: actions/download-artifact@v2
+        with:
+          name: elixir-otp-23.zip
+      - uses: actions/download-artifact@v2
+        with:
+          name: elixir-otp-24.zip
+      - uses: actions/download-artifact@v2
+        with:
+          name: Docs.zip
+      - name: Rename files
+        run: |
+          vsn=$(cat VERSION)
+          mv elixir-otp-22.zip elixir-${vsn}-otp-22.zip
+          mv elixir-otp-23.zip elixir-${vsn}-otp-23.zip
+          mv elixir-otp-24.zip elixir-${vsn}-otp-24.zip
+          mv Docs.zip Docs-${vsn}.zip
+      - name: Generate checksum
+        run: |
+          vsn=$(cat VERSION)
+          for algo in 1 256
+          do
+            shasum -a ${algo} elixir-${vsn}-otp-22.zip elixir-${vsn}-otp-23.zip elixir-${vsn}-otp-24.zip Docs-${vsn}.zip > "sha${algo}.txt"
```

Comment:
If we do this in the workflow, the checksum in Makefile is not necessary anymore?